types/foundry-vtt-api/types/apps/forms/item.d.ts

```diff
@@ -0,0 +1,49 @@
+declare interface ItemSheetData extends BaseEntitySheetData {
+    item?: Item;
+    data?: any;
+}
+
+/**
+ * The default Item Sheet
+ *
+ * This Application is responsible for rendering an item's attributes and allowing the item to be edited.
+ *
+ * System modifications may elect to override this class to better suit their own game system by re-defining the value
+ * ``CONFIG.Item.sheetClass``.
+ *
+ * @param item				The Item instance being displayed within the sheet.
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/foundry-vtt-api/types/apps/forms/item.d.ts;
  Line: 14;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.